lib/migrator.js

```diff
@@ -27,8 +27,9 @@ module.exports = class Migrator {
   }
 
   findTemplates() {
-    const templateFolderPath = path.join(this.projectRoot, 'app/templates');
-    const templateFilePaths = glob.sync(`${templateFolderPath}/**/*.hbs`);
+    const componentsFolderPath = path.join(this.projectRoot, 'app/components');
```

Comment:
In a test, the first time that the codemod ran, the component template `partials/with-partial.hbs` moved from `app/templates` to `app/components`. So, the second time it runs, I also needed to look at `app/components` to see if there are any components that use a partial.